src/main/scala/scalafix/sbt/ScalafixTestkitPlugin.scala

```diff
@@ -38,16 +38,11 @@ object ScalafixTestkitPlugin extends AutoPlugin {
       scalafixTestkitInputScalaVersion := scalaVersion.value,
       resourceGenerators.in(Test) += Def.task {
         val props = new java.util.Properties()
-        val targetrootClasspath = semanticdbOption(
-          scalafixTestkitInputScalacOptions.value,
-          "targetroot"
-        ).map(file).toSeq
         val values = Map[String, Seq[File]](
           "sourceroot" ->
             List(baseDirectory.in(ThisBuild).value),
           "inputClasspath" ->
-            (scalafixTestkitInputClasspath.value
-              .map(_.data) ++ targetrootClasspath),
+            scalafixTestkitInputClasspath.value.map(_.data),
```

Comment:
I don't understand everything, we still need to retrieve `semanticdbTargetRoot.value.toPath` no ?

Reply:
no, it's captured by parsing `scalacOptions`